src/pylir/Runtime/Support.hpp

```diff
@@ -100,6 +100,10 @@ std::intptr_t readSLEB128(const std::uint8_t** data);
     #define PYLIR_WEAK_VAR(variable, alternative) \
         variable;                                 \
         __pragma(comment(linker, "/alternatename:" #variable "=" #alternative))
+#elifdef __APPLE__
+ # define PYLIR_WEAK_VAR(variable, alternative) \
+    variable __attribute__((weak)); \
+    __asm__(".set _" #variable ", _" #alternative)
```

Comment:
Don't know if this is indeed the right way to do this...

Reply:
This whole weak attribute thing is a bit of a hack in case one wants to use LTO. I don't think LTO is that worthwhile of a use case however, so its no problem if this does not work.

Are there any tests that actually fail if this were to be a noop?